### PR TITLE
Content updates

### PR DESCRIPTION
**Pull request by desderry**

### CODE REVIEW

File: src/SFA.DAS.Campaign.Web/Views/Industries/AgricultureEnvironmentAnimalCare.cshtml [hunk 1]
```diff
@@ -8,13 +8,23 @@
 }
 
 <p class="lead-paragraph">
-    This sector can be broken down into three major categories: land management and production, animal health and welfare, and environmental industries. </p>
+    This fertile sector for apprenticeships can be separated into three major categories: 
+</p>
+<ul class="list">
```
Comment (jamesmking) [inline src/SFA.DAS.Campaign.Web/Views/Industries/AgricultureEnvironmentAnimalCare.cshtml]: I guess this should be 
```suggestion
<ol class="list list--number">
```

File: src/SFA.DAS.Campaign.Web/Views/Industries/AgricultureEnvironmentAnimalCare.cshtml [hunk 2]
```diff
@@ -8,13 +8,23 @@
...
+    <ol>1 - land management and production,</ol>
```
Comment (jamesmking) [inline src/SFA.DAS.Campaign.Web/Views/Industries/AgricultureEnvironmentAnimalCare.cshtml]: These should be 
```suggestion
    <li>1 - land management and production,</li>
```

File: src/SFA.DAS.Campaign.Web/Views/Industries/AgricultureEnvironmentAnimalCare.cshtml (hunk 1 above)
Comment (desderry) [reply src/SFA.DAS.Campaign.Web/Views/Industries/AgricultureEnvironmentAnimalCare.cshtml]: resolved for Elizabeth

File: src/SFA.DAS.Campaign.Web/Views/Industries/AgricultureEnvironmentAnimalCare.cshtml (hunk 2 above)
Comment (desderry) [reply src/SFA.DAS.Campaign.Web/Views/Industries/AgricultureEnvironmentAnimalCare.cshtml]: resolved for Elizabeth

**Pull request merged**